File: web/src/model/victron_controller/dashboard/from_0_2_0_scheduled-actions.ts
// @ts-nocheck
import {ScheduledActions as v0_2_0_ScheduledActions} from './v0_2_0/ScheduledActions'
import {ScheduledActions as dashboard_ScheduledActions} from './ScheduledActions'

export function convert__scheduled_actions__from__0_2_0(from: v0_2_0_ScheduledActions): dashboard_ScheduledActions {
    return new dashboard_ScheduledActions (
        JSON.parse(JSON.stringify(from.entries))
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_2_0_scheduled_actions.rs


pub fn convert__scheduled_actions__from__0_2_0(from: &crate::victron_controller::dashboard::v0_2_0::scheduled_actions::ScheduledActions) -> crate::victron_controller::dashboard::scheduled_actions::ScheduledActions {
    crate::victron_controller::dashboard::scheduled_actions::ScheduledActions {
        entries: serde_json::from_value(serde_json::to_value(&from.entries).unwrap()).unwrap(),
    }
}